examples/05_video_and_time.html

```diff
@@ -36,7 +36,7 @@ <h2>05 _ video and time</h2>
 
 				video = document.getElementById( 'video' );
 				video.addEventListener( 'play', function() {
-					
+
```

Comment:
Missing some `this` in this file. Are there any more files that need `this` changed?